docker/Dockerfile

```diff
@@ -21,34 +21,29 @@
 # You should have received a copy of the GNU General Public License version 2 along with
 # OpenFV. If not, see https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html.
 
-FROM openfv/openfv-base
+FROM leahmendelson/openfv:base
```

Comment:
Any reason why this had to be switched? Can we not just update `openfv/openfv-base`?

Reply:
I don't think either Berlin or I had access to push to openfv in dockerhub, but that makes more sense long term

Reply:
I just gave you access (using your hmc address) to the registry so you should be able to push now. Let me know if you want me to give you access using your gmail address instead.